runtime/src/iree/schemas/rocm_executable_def.fbs

```diff
@@ -26,6 +26,9 @@ table ExecutableDef {
   //
   block_sizes:[BlockSizeDef];
 
+  // Size of dynamic shared memory.
+  shared_memory_size:[uint32];
```

Comment:
```suggestion
  shared_memory_sizes:[uint32];
```

Reply:
just fyi (it's fine because rocm is experimental) that adding a field out of order will invalidate all existing files

Reply:
> shared_memory_sizes

Done, thanks! :)

> that adding a field out of order will invalidate all existing files

Yeah, not sure if there's a decorum on this, but for some reason the hsaco module/largest part being on the most bottom feels more correct/proper haha.

Reply:
agreed! why it's always worth adding all the fields you think you'll need from the start so they never have to be moved - or not guarantee binary compatibility so it can be changed any time :P